common/src/main/java/com/sequenceiq/cloudbreak/quartz/configuration/QuartzJobInitializer.java

```diff
@@ -34,9 +39,29 @@ public class QuartzJobInitializer {
     @Inject
     private LockService lockService;
 
-    @PostConstruct
-    private void init() {
-        lockService.lockAndRunIfLockWasSuccessful(this::initQuartz, LockNumber.QUARTZ);
+    @Inject
+    private Optional<DelayedExecutorService> delayedExecutorServiceProvider;
+
+    @Value("${quartz.initialization.delay:30}")
```

Comment:
I don't really like this approach, but it is done. (very hard to find this setting in any yaml file because of the like breaks, etc... or I haven't found a good solution to find out what is the value we set)